docs/user/dashboard/lens.asciidoc

```diff
@@ -1,14 +1,18 @@
 [[lens]]
-=== Lens
+=== Create visualizations with Lens
+++++
+<titleabbrev>Lens</titleabbrev>
+++++
 
-To create visualization panels with *Lens*, you drag the data fields you want to visualize to the workspace, then *Lens* uses heuristics to apply each field and create a visualization for you. 
+To create a visualization, drag the data fields you want to visualize to the workspace, then *Lens* uses visualization best practices to apply the fields and create a visualization that best displays the data. 
 
 With *Lens*, you can:
 
-* Create area, line, and bar charts with multiple layers, indices, and visualization types.
-* Change the aggregation function and labels to customize the data.
+* Create area, line, and bar charts with layers to display multiple indices and chart types.
+* Change the aggregation function to change how the data displays.
```

Comment:
Probably the previous version here was better.
Changing the `aggregating function` changes the data, not its representation.